config/initializers/session_store.rb

```diff
@@ -1,5 +1,12 @@
 # Be sure to restart your server when you modify this file.
 
+if Rails.env.production?
+  Rails.application.config.session_store(:cookie_store,
+                                         key:    '_turing_session',
+                                         domain: "jobs-turing.herokuapp.com")
+
+else
 Rails.application.config.session_store :cookie_store,
-  key:    '_turing_session',
-  domain: :all
+                                       key:    '_turing_session',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.